account_invoice_inter_company/views/account_move_views.xml

```diff
@@ -13,4 +13,35 @@
             </xpath>
         </field>
     </record>
+
+    <record id="view_move_form_inherit" model="ir.ui.view">
+        <field name="name">account_move_view_form_inherit</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page string="Intercompany Info" name="intercompany_info">
```

Comment:
Sorry for not detecting this on time: please hide this tab if no related bill (which is the most common case).

Reply:
ok